airbyte-cdk/python/airbyte_cdk/sources/declarative/interpolation/macros.py

```diff
@@ -61,7 +61,14 @@ def timestamp(dt: Union[numbers.Number, str]):
     if isinstance(dt, numbers.Number):
         return int(dt)
     else:
-        return int(parser.parse(dt).replace(tzinfo=datetime.timezone.utc).timestamp())
```

Comment:
use `isoparse` instead of `parse` because the valid formats are more predictable.